committeeoversightapp/templates/hearing_detail.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'base.html' %}
+{% load committeeoversight_tags %}
+
+{% block content %}
+<div class="row justify-content-center">
+  <div class="col col-lg-11">
+  <div class="hearing-form card">
+    <h2 class="text-center">{{hearing.name|title }}</h2>
+    <br />
+
+    <table class="table table-striped table-bordered" id="hearing-table">
+     <thead>
+     </thead>
+     <tbody>
+       <tr>
+         <td><b>Hearing Date</b></td>
+         <td>{{hearing.start_date}}</td>
+       </tr>
+       <tr>
+         <td><b>Committees</b></td>
+         <td>
+           {% for committee in committees %}
+             {% if committee|is_subcommittee %}
+                <a href="{{ committee.parent|get_committee_url }}">
+                  {{committee.parent}}</a>, {{committee}}
+              {% else %}
+                <a href="{{ committee|get_committee_url }}">{{committee}}</a>
+             {% endif %}
+             <br />
+            {% endfor %}
+         </td>
+       </tr>
+       <tr>
+         <td><b>Category</b></td>
+         <td>
+          {% if category|get_category_url %}
+            <a href="{{ category|get_category_url }}">{{category}}</a>
+          {% else %}
+            {{category}}
```

Comment:
Seems like `category` can be `None`, which will display the string `None` on the page, so we'll want to give this a fallback:

```suggestion
            {{category|default_if_none:""}}
```

<img width="845" alt="Screen Shot 2019-11-06 at 5 04 35 PM" src="https://user-images.githubusercontent.com/14170650/68345672-88103480-00b7-11ea-965b-9f3532395585.png">

Reply:
I'm intentionally displaying None here. All hearings should have categories, so I wanted to make clear something is missing.

Reply:
Ah, OK. Carry on then!